debian/bitlbee-common.preinst

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
+
+NAME=bitlbee
+PIDFILE=/var/run/$NAME.pid
+
+# git revision 7949d5a introduced dh_systemd.
+DH_SYSTEMD_REV="3.4.2+20160612+master+41-g7949d5a-git"
```

Comment:
What I mean is, dh_systemd being installed on a users' system doesn't say *anything* about whether they are running systemd. dh_systemd is a developer tool. It will be installed on some systems that are using sysvinit and won't be installed on most systems that are running systemd.